yb-client-wrapper/src/main/java/CmdLineParams.java

```diff
@@ -0,0 +1,54 @@
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Options;
+
+public class CmdLineParams {
+    String masterAddresses = "127.0.0.1:7100";
+    String tableName = "";
+    String dbName = "yugabyte";
+
+    String certFilePath = null;
+
+
+    public static CmdLineParams createFromArgs(String[] args) {
+        Options options = new Options();
+
+        options.addOption("master_addresses", true, "Addresses of the master process");
+        options.addOption("table_name", true, "Any table name in the database");
+        options.addOption("db_name", true, "Database for which stream needs to be created");
+        options.addOption("ssl_cert_file", true, "path to certificate file in case of SSL enabled");
```

Comment:
AsyncYBClientBuilder seems to have two methods:
- sslCertFile(String certFile)
- sslClientCertFiles(String certFile, String keyFile)

We should ideally support both.
I'm okay if we want to do this later, ssl support will need testing, but in that case, pls create a jira and tag it under 1.5!

Reply:
Adding TODO in the jar and will create the ticket as well

Reply:
Adding a TODO and also will file a ticket.